cocotb/share/lib/vpi/VpiTrampoline.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright cocotb contributors
+// Licensed under the Revised BSD License, see LICENSE for details.
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "VpiImpl.h"
+#include <cinttypes>
+#include <Windows.h>
+
+/* Define function pointers for routines */
```

Comment:
Can you expand this comment a bit explaining what this file actually does? It's a rather niche setup we have here, and I'm sure many people will be confused why we have such an "interesting" machinery in here. (Or add more documentation to the `VpiTrampoline` class below.)